### PR TITLE
fix(ui): Uniform-size checkboxes, vertically-aligned form labels, and fixes for other UI imperfections/inconsistencies

### PR DESCRIPTION
**Pull request by TheCatLady**
#### Description

Makes checkbox sizes uniform, and vertical-aligns form labels better to their fields.

Adjusts headings to be the same size and font weight.

Also reduces the horizontal space between labels & their fields, and fixes a couple of labels which I noticed didn't have the correct htmlFor attribute.

#### Screenshot (if UI related)

image

#### Todos

- [x] Successful build
- [x] Formatted General Settings page
- [x] Formatted Plex page
- [x] Formatted Services page
- [x] Formatted Notifications page
- [x] Formatted Edit User page
- [x] Formatted About page
- [x] Create reusable classes in `globals.css`

### CODE REVIEW

**Comment by sct** [comment]
How does this look on mobile?

**Comment by TheCatLady** [comment]
@sct I'm still making changes and testing.  Will attach more screenshots when I'm closer to being done

File: src/styles/globals.css [hunk 1]
```diff
@@ -34,7 +34,75 @@ body {
...
+
+.text-label {
+  @apply block text-sm font-medium leading-5 text-gray-400 sm:mt-2 mb-1
+}
+
+.group {
+  @apply mt-10 text-white
+}
+
+#group-label {
```
Comment (sct) [inline src/styles/globals.css:104]: Let's not use an id selector here.

Comment (TheCatLady) [reply src/styles/globals.css:104]: Ah I meant to change that, thanks!  I'm still fixing stuff after rebasing, but will push my changes again before I stop for the night.

File: src/components/RequestModal/AdvancedRequester/index.tsx [hunk 2]
```diff
@@ -198,7 +198,7 @@ const AdvancedRequester: React.FC<AdvancedRequesterProps> = ({
     <>
       <div className="flex items-center mb-2 font-bold tracking-wider">
         <svg
-          className="w-4 h-4 mr-1"
+          className="w-6 h-6 mr-1"
```
Comment (sct) [inline src/components/RequestModal/AdvancedRequester/index.tsx:201]: Why did we change the icon size here?

File: src/components/RequestBlock/index.tsx [hunk 3]
```diff
@@ -174,7 +174,7 @@ const RequestBlock: React.FC<RequestBlockProps> = ({ request, onUpdate }) => {
                 disabled={isUpdating}
               >
                 <svg
-                  className="w-4 h-4"
+                  className="w-6 h-6"
```
Comment (sct) [inline src/components/RequestBlock/index.tsx:177]: All the icon size changes in this file should be reverted.

File: src/components/RequestCard/index.tsx [hunk 4]
```diff
@@ -183,7 +183,7 @@ const RequestCard: React.FC<RequestCardProps> = ({ request }) => {
                   onClick={() => modifyRequest('decline')}
                 >
                   <svg
-                    className="w-4 h-4 mr-0 sm:mr-1"
+                    className="w-6 h-6 mr-0 sm:mr-1"
```
Comment (sct) [inline src/components/RequestCard/index.tsx:186]: All the icon size changes in this file should be reverted.

File: src/components/RequestList/RequestItem/index.tsx [hunk 5]
```diff
@@ -159,7 +159,7 @@ const RequestItem: React.FC<RequestItemProps> = ({
                   : `/tv/${requestData.media.tmdbId}`
               }
             >
-              <a className="min-w-0 mr-2 text-xl text-white truncate hover:underline">
+              <a className="min-w-0 mr-2 text-base truncate hover:underline">
```
Comment (sct) [inline src/components/RequestList/RequestItem/index.tsx:162]: Why did we make this smaller?

File: src/components/RequestList/RequestItem/index.tsx [hunk 6]
```diff
@@ -338,7 +338,7 @@ const RequestItem: React.FC<RequestItemProps> = ({
                   onClick={() => setShowEditModal(true)}
                 >
                   <svg
-                    className="w-4 h-4 mr-0 sm:mr-1"
+                    className="w-6 h-6 mr-0 sm:mr-1"
```
Comment (sct) [inline src/components/RequestList/RequestItem/index.tsx:341]: All the icon size changes in this file should be reverted.

File: src/components/RequestList/index.tsx [hunk 7]
```diff
@@ -54,10 +54,10 @@ const RequestList: React.FC = () => {
 
   return (
     <>
-      <div className="flex flex-col justify-between md:items-end md:flex-row">
+      <div className="flex flex-col justify-between sm:flex-row mt-8">
```
Comment (sct) [inline src/components/RequestList/index.tsx:57]: This change (among others here) broke mobiles layout

File: src/components/RequestBlock/index.tsx (hunk 3 above)
Comment (TheCatLady) [reply src/components/RequestBlock/index.tsx:177]: Yeah sorry, those were a mistake from when I was replacing the checkbox sizes!  Combing through to find them all now.

File: src/components/Settings/SettingsServices.tsx [hunk 8]
```diff
@@ -283,10 +283,11 @@ const SettingsServices: React.FC = () => {
...
               ))}
-              <li className="h-32 col-span-1 border-2 border-gray-400 border-dashed rounded-lg shadow sm:h-32">
+              <li className="h-46 col-span-1 border-2 border-gray-400 border-dashed rounded-lg shadow">
                 <div className="flex items-center justify-center w-full h-full">
                   <Button
                     buttonType="ghost"
+                    className="mt-3 mb-3"
                     onClick={() =>
                       setEditRadarrModal({ open: true, radarr: null })
                     }
```
Comment (sct) [inline src/components/Settings/SettingsServices.tsx:293]: Why did we change this?

Before:
image


Now:
image

File: src/components/Settings/SettingsAbout/Releases/index.tsx [hunk 9]
```diff
@@ -100,7 +100,7 @@ const Release: React.FC<ReleaseProps> = ({
         </Modal>
       </Transition>
       <div className="flex items-center justify-center mb-4 sm:mb-0 sm:justify-start">
-        <span className="mr-2 text-sm">
+        <span className="mr-2 text-xs mt-1">
```
Comment (sct) [inline src/components/Settings/SettingsAbout/Releases/index.tsx:103]: The changes in both this AND the Sonarr modal break the design of the modals. The text boxes are no longer full width.

File: src/components/RequestModal/AdvancedRequester/index.tsx (hunk 2 above)
Comment (sct) [reply src/components/RequestModal/AdvancedRequester/index.tsx:201]: Also, I prefer to use the older color scheme for the dropdowns in this component.

**Comment by sct** [comment]
All the inputs in the Radarr/Sonarr modals no longer fill the modal
image

**Comment by sct** [comment]
Actually, all of the labels on forms seem to have shrunk

This PR
image

Develop branch (same res)
image

File: src/components/RequestList/RequestItem/index.tsx [hunk 10]
```diff
@@ -216,7 +216,7 @@ const RequestItem: React.FC<RequestItemProps> = ({
           {requestData.modifiedBy ? (
             <span className="text-sm text-gray-300">
               {requestData.modifiedBy.displayName}
-              (
+              &nbsp;(
```
Comment (sct) [inline src/components/RequestList/RequestItem/index.tsx:219]: We should instead use padding or margin on a span element for spacing

**Comment by sct** [comment]
The request list table is cutting off content that it wouldn't before

this PR
image

Same res develop
image

**Comment by sct** [comment]
We are missing a line here

image

**Review by sct — APPROVED** [review]
💯

**Pull request merged**